core/src/main/resources/oracle/weblogic/deploy/aliases/category_modules/DomainInfo.json

```diff
@@ -11,6 +11,7 @@
         "ServerGroupTargetingLimits": [ {"version": "[10,)", "wlst_mode": "both", "wlst_name": "ServerGroupTargetingLimits", "wlst_path": "WP001", "default_value": null,    "wlst_type": "dict" } ],
         "DynamicClusterServerGroupTargetingLimits": [ {"version": "[12.2.1.1,)", "wlst_mode": "both", "wlst_name": "dynamicClusterServerGroupTargetingLimits", "wlst_path": "WP001", "default_value": null,    "wlst_type": "dict" } ],
         "ServerStartMode":            [ {"version": "[10,)", "wlst_mode": "both", "wlst_name": "ServerStartMode",            "wlst_path": "WP001", "default_value": null,    "wlst_type": "string" } ],
+        "UseSampleDatabase":          [ {"version": "[10,)", "wlst_mode": "both", "wlst_name": "UseSampleDatabase",          "wlst_path": "WP001", "default_value": null,    "wlst_type": "string" } ],
```

Comment:
I do not believe that this is applicable to 10.3.x or even 12.1.x. I think the compact topology profile was added in 12.2.x so I suspect that this option isn't really applicable before 12.2.1, though we should verify.

Reply:
The JRF-Compact typedef file is currently marking anything before 12.2.1 as NOT_SUPPORTED

Reply:
Also, I don't think this is applicable to online since it controls which database a JRF (or upper stack) domain's DataSources point to...which is only applicable prior to the domain being created.

Reply:
there are a few that way, can you setOptions after createDomain? What about ServerStartMode